docs/modules/plugins/partials/plugin-web-app-steps.adoc

```diff
@@ -84,6 +84,63 @@ Given I am on main application page
 When I go to relative URL `/products/new`
 ----
 
+==== Refresh the page
+
+Refreshes the page. Browser reloads current page which is the same action as one caused by pressing:
+
+- `F5` for most browsers
+- `Ctrl + R` for Windows
+- `Command + R` for Mac
+
+[source,gherkin]
+----
+When I refresh page
+----
+
+[WARNING]
+====
+This syntax is deprecated and will be removed
```

Comment:
please follow the same template as for other steps

Reply:
changed according to examples from generic-ui-steps.adoc